packages/earljs/src/validators/toBe.ts

```diff
@@ -0,0 +1,20 @@
+import { Control, formatValue } from './common'
+import { smartEq } from './smartEq'
+
+export function toBe<T>(control: Control<T>, expected: T) {
+  const smartEqComparisonResult = smartEq(control.actual, expected).result === 'success'
+  const strictComparisonResult = Object.is(control.actual, expected)
+
+  const additionalInfo =
```

Comment:
no test for this?

Reply:
It's tested here: https://github.com/earl-js/earl/pull/94/files#diff-70693208f147155ee9b00f0dc9aef5d47fe4cc165a14ad720c41c5d72b71082fR18

I might extract it to a separate test case...